src/makeInfinityAdder.js

```diff
@@ -6,7 +6,22 @@
  */
 
 function makeInfinityAdder() {
-  // write code here
+  let sum = 0;
+
+  function adder(value) {
+    if (value === undefined) {
+      const result = sum;
+
+      sum = 0;
+
+      return result;
+    }
+    sum += value;
```

Comment:
The sum is calculated using a simple addition. It should be calculated using either 'forEach' or 'reduce' methods as per the task requirements.